assets/src/phoenix/app.ts

```diff
@@ -134,7 +134,13 @@ document.addEventListener('DOMContentLoaded', () => {
     }),
   );
 
-  (window as any).hljs.highlightAll();
+  const hljs = (window as any).hljs;
+
+  hljs.configure({
```

Comment:
HLJS was trying to format our markdown code, so I made it much more targeted to the torus code blocks.

If it was supposed to be highlighting other things, this will introduce a bug, so need to watch for that.